src/sidebar/components/test/MarkdownEditor-test.js

```diff
@@ -154,10 +156,19 @@ describe('MarkdownEditor', () => {
               const button = wrapper.find(
                 `ToolbarButton[title="${command}"] > IconButton`
               );
+              let buttonTitlePattern;
+              if (command === 'Numbered list') {
+                // Numbered list command has extra modifier for MacOS to avoid
+                // conflict with file-open in Safari
+                buttonTitlePattern = new RegExp(
+                  `${test.expectedNumlistModifier}-${key.toUpperCase()}`
+                );
+              } else {
+                buttonTitlePattern = new RegExp(
+                  `${test.expectedModifier}-${key.toUpperCase()}`
+                );
+              }
 
-              const buttonTitlePattern = new RegExp(
-                `${test.expectedModifier}-${key.toUpperCase()}`
-              );
               assert.match(button.props().title, buttonTitlePattern);
```

Comment:
Hmm... does this actually need to be a regex or could it just check for a substring match?

Reply:
Probably not; was c-n-p-ing the structure of the existing test.